tests/auto_inject/utils.py

```diff
@@ -37,14 +39,31 @@ def _test_install(self, virtual_machine, profile: bool = False):
             logger.info(f"Http request done with uuid: [{request_uuid}] for ip [{vm_ip}]")
         wait_backend_trace_id(request_uuid, 120.0, profile=profile)
 
+    def warmup(self, virtual_machine):
+        vm_ip = virtual_machine.get_ip()
+        vm_port = virtual_machine.deffault_open_port
+        warmup_weblog(f"http://{vm_ip}:{vm_port}/")
+
+    def get_child_pids(self, virtual_machine) -> str:
```

Comment:
why not add this methods to the weblog interface?

Reply:
Done